pkg/operator/staticpod/controller/backingresource/backing_resource_controller_test.go

```diff
@@ -0,0 +1,101 @@
+package backingresource
+
+import (
+	"testing"
+	"time"
+
+	"k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes/fake"
+	ktesting "k8s.io/client-go/testing"
+
+	operatorv1alpha1 "github.com/openshift/api/operator/v1alpha1"
+	"github.com/openshift/library-go/pkg/operator/staticpod/controller/common"
+)
+
+func TestNewBackingResourceController(t *testing.T) {
+	kubeClient := fake.NewSimpleClientset()
+	var (
+		serviceAccountCreated     *v1.ServiceAccount
+		clusterRoleBindingCreated *rbacv1.ClusterRoleBinding
+		createCallCount           int
+	)
+	kubeClient.PrependReactor("*", "*", func(action ktesting.Action) (handled bool, ret runtime.Object, err error) {
```

Comment:
rather than modify the reactor, how about looking at the actions and comparing them against expected verb, resource?

Reply:
> rather than modify the reactor, how about looking at the actions and comparing them against expected verb, resource?

like this https://github.com/openshift/service-serving-cert-signer/blob/master/pkg/controller/apiservicecabundle/apiservice_cabundle_controller_test.go#L24